pom.xml

```diff
@@ -1465,6 +1469,20 @@
     <!-- Profiles -->
     <profiles>
         <!--
+          Name: modularizedJdk
+          Descr: various workarounds activation for modularized JDK
+        -->
+        <profile>
+            <id>modularizedJdk</id>
+            <activation>
+                <jdk>9</jdk>
+            </activation>
+            <properties>
+                <modular.jdk.args>-XaddExports:java.security.sasl/com.sun.security.sasl.digest=ALL-UNNAMED,java.security.sasl/com.sun.security.sasl=ALL-UNNAMED</modular.jdk.args>
+                <modular.jdk.props> -Dsun.util.logging.disableCallerCheck=true</modular.jdk.props>
```

Comment:
we should have jira for this, so it is properly addressed in jboss logging when time comes.